File: android/app/src/main/java/com/locationtracker/features/nativegoogle/models/PlaceholderPostRequest.kt
package com.locationtracker.features.nativegoogle.models

data class PlaceholderPostRequest(
    val userId: Int,
    val title: String,
    val body: String
)

data class PlaceholderPostPatchRequest(
    val title: String?,
    val body: String?,
    val userId: Int?
)

File: ios/LocationTracker/Features/NativeGoogle/Models/PlaceholderPostRequest.swift
import Foundation

/// Body model for POST and PUT. Required fields make full-create/full-replace explicit.
struct PlaceholderPostRequest: Encodable, Equatable {
  let userId: Int
  let title: String
  let body: String
}

/// Body model for PATCH. Optional fields mean "send only what changed".
struct PlaceholderPostPatchRequest: Encodable, Equatable {
  let title: String?
  let body: String?
  let userId: Int?
}
